src/services/billing-services/load-current-invoice-service.js

```diff
@@ -61,7 +61,7 @@ const adapt = (httpResponse) => {
     }
   })
   return {
-    body: parseInvoice?.length > 0 ? parseInvoice[0] : null,
+    body: parseInvoice?.length ? parseInvoice[0] : {},
```

Comment:
Com o ajuste de comentário acima, poderíamos fazer algo assim: 
```suggestion
    const { data, statusCode } = httpResponse.body
    const [invoce] = data.billDetail || []
    const parseInvoice = invoce ? { ... } : null
    return { body: parseInvoice, statusCode }    
```
Código ilustrativo